pages.ko/common/apropos.md

```diff
@@ -0,0 +1,15 @@
+# apropos
+
+> 메뉴얼 페이지에서 검색, 예를 들어 새로운 명령어 검색.
+
+- 키워드 검색:
+
+`apropos {{regular_expression}}`
```

Comment:
Perhaps `regular_expression` could be translated here too? Same for the rest of pages.

Also, don't worry about the comment above. It was me getting confused at your multiple PRs containing multiple translated pages, and being unable to tell the difference between them :P

Reply:
Of course i can do that. And thank you. In the future, when I leave many PR, I will change title a little.